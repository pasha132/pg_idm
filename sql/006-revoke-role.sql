begin;

create role test_group;
create role test_role;

insert into idm.auth_members (roleid, member)
values ('test_group'::regrole, 'test_role'::regrole);

select roleid::regrole::text, member::regrole::text, admin_option
from idm.auth_members where member = 'test_role'::regrole;

delete from idm.auth_members where member = 'test_role'::regrole;

select roleid::regrole::text, member::regrole::text, admin_option
from idm.auth_members where member = 'test_role'::regrole;

rollback;