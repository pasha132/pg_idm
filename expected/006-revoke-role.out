begin;
create role test_group;
create role test_role;
insert into idm.auth_members (roleid, member)
values ('test_group'::regrole, 'test_role'::regrole);
select roleid::regrole::text, member::regrole::text, admin_option
from idm.auth_members where member = 'test_role'::regrole;
   roleid   |  member   | admin_option 
------------+-----------+--------------
 test_group | test_role | f
(1 row)

delete from idm.auth_members where member = 'test_role'::regrole;
select roleid::regrole::text, member::regrole::text, admin_option
from idm.auth_members where member = 'test_role'::regrole;
 roleid | member | admin_option 
--------+--------+--------------
(0 rows)

rollback;
